### DISCUSSION TITLE
Release 2022.1

### DISCUSSION THREAD

**Discussion by seanbudd** [nvaccess/nvda#13724] [General]
Highlights and download links can be found in the release blog post at: https://www.nvaccess.org/post/nvda-2022-1/

<hr /><em>This discussion was created from the release <a href='https://github.com/nvaccess/nvda/releases/tag/release-2022.1'>Release 2022.1</a>.</em>

**Comment by cary-rowen** [comment]
Hi @seanbudd 
The link below is actually the release blog page for 2022.1 beta1, please correct it.
https://www.nvaccess.org/post/nvda-2022-1/

**Reply by seanbudd** [reply]
fixed, thank you (you may need to refresh your browser cache)

**Comment by cary-rowen** [comment]
The correct link is: https://www.nvaccess.org/post/release-2022-1/

**Reply by seanbudd** [reply]
That URL was set incorrectly
